src/test/java/com/zufar/icedlatte/favorite/converter/FavoriteListDtoConverterTest.java

```diff
@@ -0,0 +1,79 @@
+package com.zufar.icedlatte.favorite.converter;
+
+import com.zufar.icedlatte.favorite.dto.FavoriteListDto;
+import com.zufar.icedlatte.favorite.entity.FavoriteItemEntity;
+import com.zufar.icedlatte.favorite.entity.FavoriteListEntity;
+import com.zufar.icedlatte.openapi.dto.ProductInfoDto;
+import com.zufar.icedlatte.product.entity.ProductInfo;
+import com.zufar.icedlatte.user.stub.UserDtoTestStub;
+import org.junit.jupiter.api.BeforeEach;
+import org.mapstruct.factory.Mappers;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.DisplayName;
+
+import java.math.BigDecimal;
+import java.time.OffsetDateTime;
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class FavoriteListDtoConverterTest {
+
+    private FavoriteListDtoConverter converter;
+
+    @BeforeEach
+    void setup() {
+        converter = Mappers.getMapper(FavoriteListDtoConverter.class);
+    }
+
+    @Test
+    @DisplayName("Convert FavoriteListEntity to FavoriteListDto")
+    void convertListEntityToDto() {
+
+        ProductInfo productInfo = new ProductInfo();
+        productInfo.setProductId(UUID.randomUUID());
+
+        FavoriteItemEntity favoriteItem = new FavoriteItemEntity();
+        favoriteItem.setProductInfo(productInfo);
+
+        FavoriteListEntity expectedFavoriteListEntity = FavoriteListEntity.builder()
+                .id(UUID.randomUUID())
+                .user(UserDtoTestStub.createUserEntity())
+                .favoriteItems(Set.of(favoriteItem))
+                .updatedAt(OffsetDateTime.now())
+                .build();
+
+        FavoriteListDto actualFavoriteListDto = converter.toDto(expectedFavoriteListEntity);
+
+        assertThat(actualFavoriteListDto.id()).isEqualTo(expectedFavoriteListEntity.getId());
+        assertThat(actualFavoriteListDto.updatedAt()).isEqualTo(expectedFavoriteListEntity.getUpdatedAt());
+        assertThat(actualFavoriteListDto.userId()).isEqualTo(Optional.of(expectedFavoriteListEntity.getUser()).get().getId());
+        // fIXME: how to test favoriteItems?
```

Comment:
You could just compare list items, e.g.:

```java
var actualListitem = actualFavoriteListDto.favoriteItems().get(0);
assertEquals(favoriteItem, actualListitem);
```

Reply:
You can't get element by index from actualFavoriteListDto.favoriteItems() as it is a Set. Also these elements won't be equal as they have different types.